reference-tables.html

```diff
@@ -12,6 +12,8 @@ <h1>Reference Tables</h1>
   <li><a href="#il-senate">IL State Senate Districts</a></li>
 </ul>
 
+<p>Note, all values are in kilowatts (kW). Columns with a <code>PL_</code> prefix are planned and not yet energized projects.</p>
```

Comment:
Minor, but wanted to mention you can add `thead { position: sticky; top: 0; }` to this table to have the headers and filters stick, which makes it a lot easier to read and understand

![Screenshot from 2024-05-17 15-05-26](https://github.com/chihacknight/il-solar-map/assets/3187531/e1479666-a6ac-4c3e-8107-d049656f8b21)

The column titles are also a little technical, I don't know what `CS` means, and I think using titlecase and human words helps:

![Screenshot from 2024-05-17 15-08-51](https://github.com/chihacknight/il-solar-map/assets/3187531/9938f430-930a-4643-9408-89d258f3ddc0)

Lastly **please add units!** Otherwise it's unclear what each thing is referencing, and I thought it was the _count_ of projects, not `kW` of capacity

Reply:
adding that is a bit more involved if we want the sorting to work, so i'll open a separate issue